sdk/keyvault/azure-security-test-keyvault-jca/src/test/java/com/azure/security/keyvault/jca/AuthClientTest.java

```diff
@@ -13,19 +13,19 @@
 /**
  * The JUnit test for the AuthClient.
  */
-@Disabled
+@EnabledIfEnvironmentVariable(named = "AZURE_KEYVAULT_CERTIFICATE_NAME", matches = ".*")
 public class AuthClientTest {
 
     /**
      * Test getAuthorizationToken method.
-     * 
+     *
      * @throws Exception when a serious error occurs.
      */
     @Test
     public void testGetAuthorizationToken() throws Exception {
-        String tenantId = System.getProperty("azure.keyvault.tenant-id");
-        String clientId = System.getProperty("azure.keyvault.client-id");
-        String clientSecret = System.getProperty("azure.keyvault.client-secret");
+        String tenantId = System.getenv("AZURE_KEYVAULT_TENANT_ID");
+        String clientId = System.getenv("AZURE_KEYVAULT_CLIENT_ID");
+        String clientSecret = System.getenv("AZURE_KEYVAULT_CLIENT_SECRET");
         AuthClient authClient = new AuthClient();
         String result = authClient.getAccessToken(
```

Comment:
The propety of azure.keyvault.aad-authentication-url is now deleted, please refer to KeyVaultClient.java for the correct usage of getAccessToken function.

Reply:
TODO: this IT needs to be refactored.

Reply:
why is it still here?